scripts/zones/Western_Adoulin/npcs/Ruth.lua

```diff
@@ -37,6 +37,7 @@ end;
 function onEventFinish(player,csid,option)
     if (csid == 2523) then
         -- Progresses Quest: 'A Pioneers Best (Imaginary) Friend'
+        player:delStatusEffectsByFlag(dsp.effectFlag.INFLUENCE, true)
```

Comment:
Whoops. Sorry.

Reply:
YOU CALM DOWN

Did you have a question?

Reply:
Nope, just apologizing for you having to fix my oversights!